ASP.NET Core/Web/AspNetCoreTemplate.Web/Areas/Identity/Pages/Account/Manage/ResetAuthenticator.cshtml.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace AspNetCoreTemplate.Web.Areas.Identity.Pages.Account.Manage
+{
+    using System.Threading.Tasks;
+
+    using AspNetCoreTemplate.Data.Models;
+    using Microsoft.AspNetCore.Identity;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.RazorPages;
+    using Microsoft.Extensions.Logging;
+
+#pragma warning disable SA1649 // File name should match first type name
+    public class ResetAuthenticatorModel : PageModel
+#pragma warning restore SA1649 // File name should match first type name
+    {
+        private readonly UserManager<ApplicationUser> userManager;
+
```

Comment:
no blank lines between private readonly fields (and search for other places with this behavior)